hugegraph-core/src/main/java/org/apache/hugegraph/util/StringEncoding.java

```diff
@@ -92,33 +96,21 @@ public static int getAsciiByteLength(String value) {
     }
 
     public static byte[] encode(String value) {
-        try {
-            return value.getBytes("UTF-8");
-        } catch (UnsupportedEncodingException e) {
-            throw new HugeException("Failed to encode string", e);
-        }
+        return value.getBytes(UTF_8);
```

Comment:
please don't change the logic

Reply:
> please don't change the logic
it's recommended by the lint, and here is the differ:
![image](https://user-images.githubusercontent.com/17706099/217820320-55022b78-a771-41ac-beec-6b5c9d8dc774.png)

Reply:
ok